Add information for the workshop (#354)
precice/precice.github.io: pages/community/precice-workshop-2024.md

Keep watching this space for updates on registration, the workshop program, and more.

### Call for contributions

{% note %}
**The call for contributions to the 2024 preCICE workshop is now open!** Use this form TODO to submit your abstract for a 20-minute talk and/or a poster.
{% endnote %}

A contribution to the workshop is not necessary for participating in the workshop.

### Registration

The registration for the workshop will open in the next few weeks. You will be able to register for the full workshop or for the main part only, in case you already know the topics covered by the course or you cannot attend all days.

### Registration fee

#### Early-bird registration

| Ticket type   | Members of academia | Members of industry  |
|--------------|-----------|------------|
| **full registration** | 300 Euro    | 500  Euro     |

> | **main part only**      | 200 Euro  | 300 Euro       |

Reviewer: i find "main part only" a bit odd. couldn't we be explicit here? "course + workshop" and "workshop". or "days 1-4", "days 2-4"?

Author: i replaced with "workshop" and "course+workshop"

Reviewer 2: That could lead to confusions later on, when we always refer to it as "preCICE Workshop", but then people see "course + workshop".

Author: what about "workshop with course" and "workshop without course"?

Reviewer 2: Long, but clear. We can stick to that for now, and something better might pop up later. :+1:

<img class="img-responsive center-block" src="images/events/precice2024.svg" alt="preCICE Workshop banner" style="max-width: 500px; width: 100%; margin:auto;">

The 5th preCICE Workshop will be held at the [University of Stuttgart](https://www.uni-stuttgart.de/), together with [SimTech](https://www.simtech.uni-stuttgart.de/), on September 24-27, 2024. The workshop is a coming together of the preCICE community to share ideas, experiences and knowledge about using preCICE, and to learn from others in the process. Like always, we plan to have user and developer talks, hands-on training sessions, discussions with the developers on your applications and use cases, and plenty of opportunities for networking. Besides, this year we will organize a preCICE poster session for the first time!

> The workshop will start with a course on September 24-25. The [course](https://precice.org/community-training.html) is suited for both beginners and non-beginners, since advanced topics will also be covered. The main part of the workshop will take part on September 25-27. A detailed program for the workshop will be provided later on. 

Reviewer 2: ```suggestion
The workshop will start with a hands-on training course on September 24-25. The [course](https://precice.org/community-training.html) is suited for both beginners and current preCICE users, since advanced topics will also be covered. The main part of the workshop will take part on September 25-27. A detailed program for the workshop will be provided later on. 
```

Keep watching this space for updates on registration, the workshop program, and more.

### Call for conributions

> > **The call for contributions to the **2024 preCICE workshop** is now open! Use this form TODO to submit your abstract for a 20-minute talk and/or a poster.**

Reviewer 2: Instead of quotes and boldface, better use one of the existing alerts:

https://precice.org/docs-meta-cheatsheet.html

Author: Great, it's updated now:)

A contribution to the workshop is not necessary for participating in the workshop. 

### Registration

> The registration for the workshop will open in the next few weeks. You will be able to register for the full workshop or for the main part only. 

Reviewer 2: ```suggestion
The registration for the workshop will open in the next few weeks. You will be able to register for the full workshop or for the main part only, in case you already know the topics covered by the course or you cannot attend all days. 
```

> ### Registration fee

Reviewer 3: I would definitely add a price column for holders of support licenses.

For industry members, it could make sense to get a license instead of paying the normal price. Making this very prominent early on gives them the opportunity to discuss this with their employer.

Reviewer: yes, good idea. would be 0€ then. and please link to https://precice.org/community-support-precice.html#2-support-license

- **Early-bird registration** 

| Ticket type   | Members of academia | Members of industry  |

> |--------------|-----------|------------|

Reviewer 2: Not sure if it matters here, but I would suggest to always use only three dashes per column. I have seen situations where this leads to strange situations with alignment:

```suggestion
|---|---|---|
```

| **full registration** | 300 Euro    | 500  Euro     |
| **main part only**      | 200 Euro  | 300 Euro       |

- **Standard registration**

| Ticket type   | Members of academia | Members of industry  |
|--------------|-----------|------------|
| **full registration** | 450 Euro    | 750  Euro     |
| **main part only**      | 300 Euro  | 500 Euro       |

If you are a student and struggle to find funding, you might be eligible for our student travel awards. 

### Contact

> For questions related to the **2024 preCICE workshop**, write us to preCICE2024@simtech.uni-stuttgart.de.

Reviewer 2: Is highlighting important here?

```suggestion
For questions related to the 2024 preCICE workshop, write us to preCICE2024@simtech.uni-stuttgart.de.
```

<img class="img-responsive center-block" src="images/events/precice2024.svg" alt="preCICE Workshop banner" style="max-width: 500px; width: 100%; margin:auto;">

> The 5th preCICE Workshop will be held at the [University of Stuttgart](https://www.uni-stuttgart.de/), together with [SimTech](https://www.simtech.uni-stuttgart.de/), on September 24-27, 2024. The workshop is a coming together of the preCICE community to share ideas, experiences and knowledge about using preCICE, and to learn from others in the process. Like always, we plan to have user and developer talks, hands-on training sessions, discussions with the developers on your applications and use cases, and plenty of opportunities for networking. Besides, this year we will organize a preCICE poster session for the first time!

Reviewer 4: ```suggestion
The 5th preCICE Workshop will be held at the [University of Stuttgart](https://www.uni-stuttgart.de/), together with [SimTech](https://www.simtech.uni-stuttgart.de/), on September 24-27, 2024. The workshop is a coming together of the preCICE community to share ideas, experiences and knowledge about using preCICE, and to learn from others in the process. Like always, we plan to have user and developer talks, hands-on training sessions, discussions with the developers about your applications and use cases, and plenty of opportunities for networking. Besides, this year we will organize a preCICE poster session for the first time!
```

> The workshop will start with a hands-on training course on September 24-25. The [course](https://precice.org/community-training.html) is suited for both beginners and current preCICE users, since advanced topics will also be covered. The main part of the workshop will take part on September 25-27. A detailed program for the workshop will be provided later on.

Reviewer 4: ```suggestion
The workshop will start with a hands-on training course on September 24-25. The [course](https://precice.org/community-training.html) is suited for both beginners and current preCICE users, since advanced topics will also be covered. The main part of the workshop will take part on September 25-27. A detailed program for the workshop will be available soon.
```

Keep watching this space for updates on registration, the workshop program, and more.

### Call for contributions

{% note %}
The call for contributions to the 2024 preCICE workshop is now open! Use the [submission form](https://ipvscloud.informatik.uni-stuttgart.de/apps/forms/s/JYtwkfiiiMCjHGjqjACifDWy) to submit your abstract for a talk and/or a poster.
{% endnote %}

A contribution to the workshop is not necessary for participating in the workshop.

### Registration

The registration for the workshop will open in the next few weeks. You will be able to register for the full workshop or for the main part only, in case you already know the topics covered by the course or you cannot attend all days.

### Registration fee

#### Early-bird registration

| Ticket type | Members of academia | Members of industry  |
|---|---|---|
| **workshop + course** | 300 Euro    | 500  Euro     |
| **workshop**      | 200 Euro  | 300 Euro       |

#### Standard registration

| Ticket type | Members of academia | Members of industry  |
|---|---|---|
| **workshop + course** | 450 Euro    | 750  Euro     |
| **workshop**      | 300 Euro  | 500 Euro       |

> If you are a student and struggle to find funding, you might be eligible for our student travel awards.

Reviewer 4: ```suggestion
If you are a student or a researcher who has difficulties in finding funding, you might be eligible for our travel awards. More details will come soon.
```

> Holders of a [preCICE suport license](https://precice.org/community-support-precice.html#2-support-license) are excempt of the registration fee and can register to the workshop by simply confirming assistance to preCICE2024@simtech.uni-stuttgart.de.

Reviewer 4: ```suggestion
Holders of a [preCICE suport license](https://precice.org/community-support-precice.html#2-support-license) are exempt of the registration fee and can register to the workshop by simply confirming assistance to preCICE2024@simtech.uni-stuttgart.de.
```

### Contact

> For questions related to the 2024 preCICE workshop, write us to preCICE2024@simtech.uni-stuttgart.de.

Reviewer 4: ```suggestion
For questions related to the 2024 preCICE workshop, write  to us at preCICE2024@simtech.uni-stuttgart.de.
```

#### Early-bird registration

| Ticket type | Members of academia | Members of industry  |
|---|---|---|
| **workshop + course** | 300 Euro    | 500  Euro     |

> | **workshop**      | 200 Euro  | 300 Euro       |

Reviewer 2: I find the extra bolds a bit too much visually, the table already conveys the meaning without extra styling.

General good practice: separate the style from the content as much as possible.

Author: extra bolds removed

#### Standard registration

| Ticket type | Members of academia | Members of industry  |
|---|---|---|
| **workshop + course** | 450 Euro    | 750  Euro     |
| **workshop**      | 300 Euro  | 500 Euro       |

If you are a student and struggle to find funding, you might be eligible for our student travel awards.

> Holders of a [preCICE suport license](https://precice.org/community-support-precice.html#2-support-license) are excempt of the registration fee and can register to the workshop by simply confirming assistance to preCICE2024@simtech.uni-stuttgart.de.

Reviewer 2: Not sure if "assistance" is the right word here.
Still important to emphasize when they need to do that (late registration deadline, probably).
